modules/football/widget.go

```diff
@@ -0,0 +1,181 @@
+}
+
+func (widget *Widget) content() (string, string, bool) {
+
+	var content string
+	title := fmt.Sprintf("%s %s", widget.CommonSettings().Title, widget.League.caption)
+	wrap := false
+	if widget.err != nil {
+		return title, widget.err.Error(), true
+	}
+	content += widget.GetStandings(widget.League.id)
+	content += widget.GetMatches(widget.League.id)
+
+	return title, content, wrap
+}
+
+func getLeague(league string) (leagueInfo, error) {
+
+	var l leagueInfo
+	if val, ok := leagueID[league]; ok {
+		return val, nil
+	}
+	return l, fmt.Errorf("No such league")
+}
+
+// GetStandings of particular league
+func (widget *Widget) GetStandings(leagueId int) string {
+
+	var l LeagueStandings
+	var content string
+	content += fmt.Sprintf("Standings:\n\n")
+	buf := new(bytes.Buffer)
+	tStandings := createTable([]string{"No.", "Team", "MP", "Won", "Draw", "Lost", "GD", "Points"}, buf)
+	resp, err := widget.Client.footballRequest("standings", leagueId)
+	if err != nil {
+		return fmt.Sprintf("Error fetching standings: %s", err.Error())
+	}
+	defer resp.Body.Close()
+	data, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Sprintf("Error fetching standings: %s", err.Error())
+	}
+	err = json.Unmarshal(data, &l)
+	if err != nil {
+		return fmt.Sprintf("Error fetching standings")
+	}
+
+	if len(l.Standings) > 0 {
+		for _, i := range l.Standings[0].Table {
+			if i.Position <= widget.settings.standingCount {
+				row := []string{strconv.Itoa(i.Position), i.Team.Name, strconv.Itoa(i.PlayedGames), strconv.Itoa(i.Won), strconv.Itoa(i.Draw), strconv.Itoa(i.Lost), strconv.Itoa(i.GoalDifference), strconv.Itoa(i.Points)}
+				tStandings.Append(row)
+			}
+		}
+	} else {
+		return fmt.Sprintf("Error fetching standings")
+	}
+
+	tStandings.Render()
+	content += buf.String()
+
+	return content
+}
+
+// GetMatches of particular league
+func (widget *Widget) GetMatches(leagueId int) string {
+
+	var l LeagueFixtuers
+	var content string
+	scheduledBuf := new(bytes.Buffer)
+	playedBuf := new(bytes.Buffer)
+
+	tScheduled := createTable([]string{}, scheduledBuf)
+	tPlayed := createTable([]string{}, playedBuf)
+
+	from := getDateString(-widget.settings.matchesFrom)
+	to := getDateString(widget.settings.matchesTo)
+
+	requestPath := fmt.Sprintf("matches?dateFrom=%s&dateTo=%s", from, to)
+	resp, err := widget.Client.footballRequest(requestPath, leagueId)
+	if err != nil {
+		return fmt.Sprintf("Error fetching matches: %s", err.Error())
+	}
+	defer resp.Body.Close()
+	data, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Sprintf("Error fetching matches: %s", err.Error())
+	}
+	err = json.Unmarshal(data, &l)
+	if err != nil {
+		return fmt.Sprintf("Error fetching matches: %s", err.Error())
+	}
+
+	if len(l.Matches) != 0 {
+
+		for _, val := range l.Matches {
+			if strings.Contains(val.AwayTeam.Name, widget.settings.favTeam) || strings.Contains(val.HomeTeam.Name, widget.settings.favTeam) || widget.settings.favTeam == "" {
```

Comment:
Sure, that was kind of early version which I wanted to fix later on..but missed :)

As suggested, I have extracted it as a function that will mark the fav team with Star. Also, it will print reset other fixtures or past matches.

![image](https://user-images.githubusercontent.com/20225764/67507247-d290ba80-f68e-11e9-8eb4-9b5cb73f7907.png)